src/components/ui/fromField.js

```diff
@@ -0,0 +1,12 @@
+import { Field } from "formik";
+import { TextField } from "formik-mui";
+
+const FormField = (props) => {

```

Comment:
Suggestion how to improve readability with unpacking props.
We can do `({ children, ...rest })`